base/_flexbox.scss

```diff
@@ -0,0 +1,173 @@
+@mixin flexbox {
```

Comment:
Please could you add some usage docs for these mixins?

Reply:
Sure, in the readme file?

Reply:
Mmm I'd probably go in the mixins file itself, like how we document modules or php functions :)